resources/resources.go

```diff
@@ -37,19 +37,17 @@ func InstallPostgres(resourceDir, dataDir string) (string, error) {
 	}
 
 	// Skip installation if installed already.
-	pgBinDir := path.Join(resourceDir, version)
-	_, err := os.Stat(pgBinDir)
+	_, err := os.Stat(dataDir)
```

Comment:
I guess there is a case that we only remove data directory instead of the entire pg binary, and let PG init it again (especially in the testing scenario).

The current logic looks like we are always going to reinstall the pg binary in such case.

Reply:
Yeah, it's fine to reinstall in that rare case as long as we're not reinstalling the binary on every start.

Reply:
I am thinking this is not a rare case though especially during test environment.

We always do this this SQLite https://github.com/bytebase/bytebase/wiki/How-to-perform-upgrade#option-1---reset-the-data

Though, after we have PG, we likely we will use migration file and reduce the likelihood of using this approach.

But still, it's always a good practice to decouple data (dataDir) from code (binDir), and handle them separately.

Reply:
Yes, it's a good suggestion to decouple binary installation and data setup. Done.